app/views/shared/_slotpicker_templates.html.erb

```diff
@@ -0,0 +1,76 @@
+<script type="text/html" id="SlotPicker-tmplDay">
+  <li class="SlotPicker-day" id="date-{{ slot }}" tabindex="-1">
+    <h2 class="SlotPicker-dayTitle">{{ date }}</h2>
+    {{#if oneSlot}}
+    <p class="SlotPicker-dayOneSlot"><%= t('.one_slot_available') %></p>
+    {{/if}}
```

Comment:
![yodawg](https://cloud.githubusercontent.com/assets/17347/11742952/b587465e-9ffb-11e5-8e2d-9052f6ea181c.jpg)

Seriously, if there is no interpolated Ruby in these files, they can just be `.html` instead of `.html.erb`.